src/components/DerivationPathField.js

```diff
@@ -36,7 +38,7 @@ export default function DerivationPathField(props) {
 		<>
 			<TouchableOpacity
 				onPress={toggleShowAdvancedField}
-				style={{ diplay: 'flex' }}
+				//style={{ diplay: 'flex' }}
```

Comment:
```suggestion
				style={{ display: 'flex' }}
```

Reply:
I think `display:flex` does not work in react native, so just delete it.